File: app/src/main/java/can/siempredelao/f1kotlin/backend/Result.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class Result {

    private String      number;
    private String      position;
    private String      positionText;
    private String      points;
    @SerializedName("Driver")
    private Driver      driver;
    @SerializedName("Constructor")
    private Constructor constructor;
    private String      grid;
    private String      laps;
    private String      status;
    @SerializedName("Time")
    private Time        time;
    @SerializedName("FastestLap")
    private FastestLap  fastestLap;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/Result.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class Result(val number: String,
             val position: String,
             val positionText: String,
             val points: String,
             @SerializedName("Driver") val driver: Driver,
             @SerializedName("Constructor") val constructor: Constructor,
             val grid: String,
             val laps: String,
             val status: String,
             @SerializedName("Time") val time: Time,
             @SerializedName("FastestLap") val fastestLap: FastestLap)
